examples/formats/NEXRAD_Level_3_File.py

```diff
@@ -17,17 +17,18 @@
 ###########################################
 fig, axes = plt.subplots(1, 2, figsize=(15, 8))
 add_metpy_logo(fig, 190, 85, size='large')
-for v, ctable, ax in zip(('N0Q', 'N0U'), ('NWSReflectivity', 'NWSVelocity'), axes):
+ctables = (('NWSStormClearReflectivity',  -20, 0.5),  # dBZ
+           ('NWS8bitVel',                -100, 1.0))  # m/s
```

Comment:
The spacing after the name is throwing off the style checkers. Can you reduce to only a single space after the `,`? I know that messes up the alignment.

Reply:
Done